serving/backend/api_models.py

```diff
@@ -2,12 +2,12 @@
 
 
 class Properties(BaseModel):
-    name: str
-    str: int
-    dex: int
-    con: int
-    wis: int
-    cha: int
-    ac: int
-    hp: int
-    int: int
+    name: "str"
+    str: "int"
+    dex: "int"
+    con: "int"
+    int: "int"
+    wis: "int"
+    cha: "int"
+    ac: "int"
+    hp: "int"
```

Comment:
Czemu? Przecież typy w Pydanticu powinny być właśnie typami Pythonowymi

Reply:
Z tego co wiem to był konflikt między nazwą i typem
Chciałyśmy, żeby nazwy były w odpowiedniej kolejności - tak jak frontend,
ale bez `" "` `int: `  musiałby być na samym końcu

Reply:
Aaa, no tak, to jest zabawny problem xD
Można użyć aliasów: https://stackoverflow.com/a/60621899/9472066. W sensie nazwać te pola `strength`, `intelligence` etc., ale dodać wszędzie aliasy. W sumie byłoby też spoko, bo od razu zwiększyłoby czytelność (np. "ac" -> "Armor Class" jest mocno dziedzinowe)